APE14.rst

```diff
@@ -0,0 +1,450 @@
+    pos.cartesian.y          | Cartesian coordinate along the y-axis
+    pos.cartesian.z          | Cartesian coordinate along the z-axis
+    pos.ecliptic.lat         | Ecliptic latitude
+    pos.ecliptic.lon         | Ecliptic longitude
+    pos.eq.dec               | Declination in equatorial coordinates
+    pos.eq.ra                | Right ascension in equatorial coordinates
+    pos.galactic.lat         | Latitude in galactic coordinates
+    pos.galactic.lon         | Longitude in galactic coordinates
+    pos.healpix              | Hierarchical Equal Area IsoLatitude Pixelization
+    pos.heliocentric         | Heliocentric position coordinate (solar system bodies)
+    spect.dopplerVeloc       | Radial velocity, derived from the shift of some spectral feature
+    spect.dopplerVeloc.opt   | Radial velocity derived from a wavelength shift using the optical convention
+    spect.dopplerVeloc.radio | Radial velocity derived from a frequency shift using the radio convention
+    time                     | Time, generic quantity in units of time or date
+    time.epoch               | Instant of time related to a generic event (epoch, date, Julian date, time stamp/tag,...)
+
+The full UCD1+ vocabulary does not include all possible type names that would be
+needed to represent **all** WCSes (for example, there are no keywords for
+helioprojective coordinates). In this case, the element of
+``world_axis_physical_types`` for those coordinates should be a string prefixed
+with ``custom:``. This should also be taken as a call to work with the
+International Virtual Observatory Alliance (IVOA) to implement
+new type names, which the Astropy Project will facilitate as needed. If a
+``custom:`` type name is needed, we recommend that these be coordinated and
+agreed as much as possible between different packages to make sure that these
+can be useful (which would not be the case if each package created their own
+set of custom type names).
+
+High-level Astropy Object
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Unlike the low-level API, the ‘high-level’ interface described here will be a
+single Astropy-developed class since it interfaces with various Astropy objects.
+This high-level API would provide the ability for example to get ``SkyCoord``,
+``Time`` etc. objects back from a pixel to world conversion, and conversely to
+be able to convert ``SkyCoord``, ``Time`` etc. to pixel values.
+
+The high-level object would not inherit from the low-level classes but instead
+wrap them. The high-level object should provide at a minimum the
+following two methods:
+
+.. code-block:: python
+
+    def pixel_to_world(self, *pixel_arrays):
+        """
+        Convert pixel coordinates to world coordinates (represented by Astropy
+        objects).
+        """
+
+    def world_to_pixel(self, *world_objects):
+        """
+        Convert world coordinates (represented by Astropy objects) to pixel
+        coordinates
+        """
```

Comment:
I suggest that this method support named arguments for the arrays instead of forcing the user to get the argument order correct. The code will be easier to read and more robust.

Also consider returning a dict or a struct with named fields (to simplify element access, e.g. `world.sky` instead of `world["sky"]`), or at least a named tuple (which I dislike because it allows -- even encourages -- access by index).